br/pkg/checksum/executor.go

```diff
@@ -330,13 +332,30 @@ func (exec *Executor) Execute(
 	updateFn func(),
 ) (*tipb.ChecksumResponse, error) {
 	checksumResp := &tipb.ChecksumResponse{}
+	checksumBackoffer := utils.NewFixedBackoffer(utils.ChecksumRetryTime, utils.ChecksumWaitInterval, utils.ChecksumMaxWaitInterval)
```

Comment:
Can we use `RetryState` here (instead of changing the definition of other backoffers)?

Reply:
good catch